pkg/translatedaudit/restriction_level.go

```diff
@@ -0,0 +1,48 @@
+package translatedaudit
+
+import (
+	"fmt"
+
+	"github.com/samber/lo"
+
+	"github.com/cmsgov/mint-app/pkg/models"
+)
+
+// checkIfDocumentIsRestricted looks at the translated fields for a document to see if it should be restricted or not
+func checkIfDocumentIsRestricted(tAuditFields []*models.TranslatedAuditField, operation models.DatabaseOperation) (bool, error) {
+
+	restrictedField, fieldFound := lo.Find(tAuditFields, func(field *models.TranslatedAuditField) bool {
+		return field.FieldName == "restricted"
+	})
+	if !fieldFound {
+		return false, fmt.Errorf("couldn't find the field to say if this document is restricted or not")
```

Comment:
We should return true that a document is restricted in the case of an error so as to protect sensitive user data

Reply:
good call,  that's also aligned with the behavior upstream for this function. Thanks! [2d5f77c](https://github.com/CMSgov/mint-app/pull/1211/commits/2d5f77cbf9947ab32a958dbd56f875b265774060)